src/test/java/com/epam/prejap/teatrees/block/TBlockTest.java

```diff
@@ -0,0 +1,87 @@
+package com.epam.prejap.teatrees.block;
+
+import com.epam.prejap.teatrees.TeaTrees;
+import com.epam.prejap.teatrees.game.Playfield;
+import com.epam.prejap.teatrees.game.Printer;
+import com.epam.prejap.teatrees.game.Waiter;
+import com.epam.prejap.teatrees.player.Player;
+import com.epam.prejap.teatrees.player.RandomPlayer;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.testng.Assert.assertEquals;
+
+@Test
+public class TBlockTest {
```

Comment:
I think that this is an integration test so in my opinion it would be better if you will annotate this class with appropriate suffix e.g. *IT.java or *IntegrationTest.java.

Also this test should not be run with mvn test scope but mvn verify.

Reply:
done